packages/core/src/pipeline/create/CreatePipelineModal.tsx

```diff
@@ -185,7 +185,7 @@ export class CreatePipelineModal extends React.Component<ICreatePipelineModalPro
       const configs: IPipeline[] = config.strategy
         ? application.strategyConfigs.data
         : application.pipelineConfigs.data;
-      const newPipeline = configs.find((_config) => _config.name === config.name);
+      const newPipeline = configs.find((_config) => _config.name === config.name.trim());
```

Comment:
I think you should do this when the pipeline is initially submitted to be created [L132](https://github.com/spinnaker/deck/blob/c909e83ac33d3975f09e0e231f75d68246cb4bef/packages/core/src/pipeline/create/CreatePipelineModal.tsx#L132).

Reply:
Updated the code.